### PR TITLE
fix: DidMethod interface impl by HTTP DID Resolver

### LINKED ISSUE #145: Adopt HTTP DID Resolver to the changed Resolver interface
Adopt HTTP DID Resolver method signature
https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didmethod/httpbinding/resolver.go#L109

to reflect the change of the Resolver interface:

https://github.com/hyperledger/aries-framework-go/blob/master/pkg/framework/didresolver/api.go#L32

### PR DESCRIPTION
**Pull request by kdimak**
Signed-off-by: Dima <dkinoshenko@gmail.com>

closes #145 

Adopts HTTP DID Resolver to implement the changed Resolver interface.

### CODE REVIEW

File: pkg/didmethod/httpbinding/resolver.go [hunk 1]
```diff
@@ -106,14 +107,19 @@ func New(endpointURL string, opts ...ResolverOpt) (*DIDResolver, error) {
 }
 
 // Read implements didresolver.DidMethod.Read interface (https://w3c-ccg.github.io/did-resolution/#resolving-input)
-func (res *DIDResolver) Read(DID string, versionID interface{}, versionTime string, noCache bool) ([]byte, error) {
+func (res *DIDResolver) Read(DID string, _ ...didresolver.ResolveOpt) ([]byte, error) {
```
Comment (llorllale) [inline pkg/didmethod/httpbinding/resolver.go]: Does the universal resolver support these options?

**Comment by llorllale** [comment]
we now have an unfortunate similarity between the symbols `httpbinding.ResolverOpt` and `didresolver.ResolveOpt`

**Comment by troyronda** [comment]
Should ResolveOpt actually be MethodOpt?

File: pkg/didmethod/httpbinding/resolver.go (hunk 1 above)
Comment (fqutishat) [reply pkg/didmethod/httpbinding/resolver.go]: we will use them in the future

Comment (kdimak) [reply pkg/didmethod/httpbinding/resolver.go]: > Does the universal resolver support these options?

Currently, there is no instruction on how to use these parameters in HTTP(s) binding in the [spec](https://w3c-ccg.github.io/did-resolution/#bindings).

**Pull request merged**